test/usage.cjs

```diff
@@ -3198,6 +3198,61 @@ describe('usage tests', () => {
     });
   });
 
+  describe('showVersion', () => {
+    // see #143.
+    it('should show version regardless of whether argv has been called', () => {
```

Comment:
Thank you for the tests 👍 

I think you just need to run `npm run fix` to address these linting issues.